docs/src/pages/en/migration/from-0.x-to-1.0.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Migrating from v0.x to v1.0
+description: Migration guide
+layout: ../../../layouts/MainLayout.astro
+---
+
+While there were some sweeping changes to Media Chrome, most of these should be fairly straightforward to refactor. Here's a high level overview:
+
+### Breaking changes
+
+**General**
+- All attributes have been changed from `kebab-case` to `lowercase` (aka "`smushedcase`")
+- Several CSS variables have been renamed for consistency, more consistently applied, and more consistently defaulted
+- `AttributeTokenList` (e.g. `hotkeys`) removed the `keys()` method
+
+**Captions & Subtitles**
+- Captions & Subtitles list attributes are now just subtitles attributes
+- Enabling captions/subtitles by default belongs to the `<media-controller>` instead of the `<media-captions-button>`
```

Comment:
would be nice to show the new attribute here so that they don't need to look it up separately.

Reply:
ah, I guess it exists below. Would be nice to have these link there.